opba-banking-rest-api-ymls/src/main/resources/static/tpp_banking_api_commons.yml

```diff
@@ -254,6 +254,54 @@ components:
       schema:
         type: string
 
+    TPP-Decoupled-Preferred:
```

Comment:
Do not use TPP prefix, use FinTech instead. ASPSP will not talk directly to our clients, instead, it will call OBG and OBG will proxy the requests with i.e. Webhook use case